packages/offix-datastore/src/storage/Storage.ts

```diff
@@ -1,55 +1,54 @@
 import { PushStream, ObservablePushStream } from "../utils/PushStream";
-import { Model, PersistedModel } from "../models";
 import { PredicateFunction } from "../predicates";
 import { createDefaultStorage } from "./adapters/defaultStorage";
 import { generateId } from "./core";
+import { EventTypes } from "../models";
 
 export interface IStorageAdapter {
```

Comment:
Every interface needs to have proper comments

Reply:
I have added some comments